catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ImportCommand.java

```diff
@@ -134,8 +145,18 @@ protected Object executeWithSubject() throws Exception {
               + "File being imported: {}",
           importFile);
     } else {
-      if (!zipValidator.validateZipFile(importFile)) {
-        throw new CatalogCommandRuntimeException("Signature on zip file is not valid");
+      if (StringUtils.isBlank(signatureFile)) {
+        throw new CatalogCommandRuntimeException(
```

Comment:
❓ Are we sure that this exception message content gets printed to the user so they can try to fix it? if not we may want to write it manually to the console.